packages/embark-api/src/server.ts

```diff
@@ -23,16 +21,14 @@ interface CallDescription {
 export default class Server {
   private _isInsideMonorepo: boolean | null = null;
   private _monorepoRootDir: string = "";
-  private embarkUiBuildDir: string = (
-    findUp.sync("node_modules/embark-ui/build", {cwd: embarkPath()}) ||
-    embarkPath("node_modules/embark-ui/build")
-  );
+  private embarkUiBuildDir: string = "";
   private expressInstance: expressWs.Instance;
   private isLogging: boolean = false;
   private server?: http.Server;
 
   constructor(private embark: Embark, private port: number, private hostname: string, private plugins: Plugins) {
     this.expressInstance = this.initApp();
+    this.embarkUiBuildDir = (findUp.sync("node_modules/embark-ui/build", {cwd: embark.fs.embarkPath()}) || embark.fs.embarkPath("node_modules/embark-ui/build"));
```

Comment:
Just a remark: It's kinda dangerous to put `node_modules/embark-ui` (specifically `embark-ui`) in different places here. Maybe we can put it at least in a constant? I wouldn't be surprised that we change `embark-ui` to `cockpit` in the future.

Reply:
@PascalPrecht I'll add this to pivotal since it's out of scope for this PR